WTWSMS/decisions/VIET_roleplay_decisions.txt

```diff
@@ -12,7 +12,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			is_landed = yes
-			ai = no
```

Comment:
Only counts with two provinces and up can go on journeys. That's a fair restriction, right? If you wish, I am willing to set it so that AI dukes and up are the only ones that can go on adventures, to reduce the lag that does not actually exist.